jsonnet/kube-prometheus/components/grafana.libsonnet

```
@@ -83,4 +83,20 @@ function(params)
         }],
       },
     },
+
+    // The override adding 'allowPrivilegeEscalation: false' can be deleted when
```

```suggestion
    // FIXME(ArthurSens): The override adding 'allowPrivilegeEscalation: false' can be deleted when
```
nit: Let's just add FIXME so it is easier to spot later ;)